web/src/components/StyledToast.js

```diff
@@ -14,7 +14,7 @@ export default function StyledToast() {
         reverseOrder={false}
         position='top-right'
         toastOptions={{
-          duration: Infinity,
+          duration: 4000,
```

Comment:
Hey @ch3yne , we should define a constant value for the debounce value for the persistent UX. Thanks!

Reply:
Users usually don't modify the frontend code, do you means that we need to add this in the Gravitino configuration file?

Reply:
Nope, it should be defined in a constant file and referred to. IMO, debounce is common and frequent in UI.

Reply:
Okay, I will add a configuration file to manage common user interface variables.

Reply:
@ch3yne Great! Thanks.